lib/lotus/loader.rb

```diff
@@ -60,6 +61,7 @@ def _configure_controller_framework!
             cookies config.cookies.default_options
           end
           prepare { include Lotus::Action::Session } if config.sessions.enabled?
+          prepare { include Lotus::Action::RoutingHelpers }
```

Comment:
@joneslee85 in this line, routing helpers are loaded for views https://github.com/lotus/lotus/blob/master/lib/lotus/generators/slice/application.rb.tt#L209